### ISSUE TITLE
Display entitlements built into the selected app

### ISSUE THREAD

**Issue by macblazer** [jamf/PPPC-Utility#107]
As an admin I would like to see the entitlements built-in to the selected app (just like seeing the code signing requirements) so that I know what the app is trying to do, and can more easily grant the correct access to functionality that I want to enable without the system prompting the end user.

**Comment by uurazzle** [comment]
New versions of [Suspicious Package](https://www.mothersruin.com/software/SuspiciousPackage/?utm_source=help-menu&utm_medium=suspicious-package-app) and [Apparency](https://www.mothersruin.com/software/Apparency/) have added features to make it a little easier to discover and create PPPC configuration profiles. Would still love a more direct integration with PPPC Uitlity.

**Comment by rsaldinger** [comment]
Yes, that's the "merged entitlements" view, described [for Suspicious Package here](https://www.mothersruin.com/software/SuspiciousPackage/faq.html#merged-entitlements). This collects all of the entitlements requested across all executables in the package, and also provides a way to copy the executable/app information to the clipboard in the "app identity" form required for the PPPC Profile. Apparency has a [parallel feature](https://www.mothersruin.com/software/Apparency/faq.html#merged-entitlements) that collects all entitlements for the app and all of the components inside it.

I'm still open to doing something more specific to work with PPPC Utility, but don't know what that might be. FWIW, I mocked up a sort of "executable audit" format for the entitlement/executable information, which can be generated using an as-yet-undocumented option on the "spkg" CLI tool for Suspicious Package v4.2, e.g.:

`spkg --exec-audit audit.plist SomePackage.pkg`

If you open up the resulting plist, I _think_ the contents should be fairly self-explanatory. Whether something like this is useful or not, I've no clue! (I also added a parallel option to the appy CLI tool for Apparency v1.4.1, but I see now that I broke it at some point thereafter; it could be made to work, though, if it's of value.)

**Comment by uurazzle** [comment]
Hi @macblazer,

Thanks for adding the entitlements to the help buttons. Is there still interest in having tighter integration integration or sharing of entitlements with tools like f [Suspicious Package](https://www.mothersruin.com/software/SuspiciousPackage/?utm_source=help-menu&utm_medium=suspicious-package-app) and [Apparency](https://www.mothersruin.com/software/Apparency/) it would make it easier to discover and create PPPC configuration profiles. Did you have any additional questions about integration options or benefits?

**Comment by macblazer** [comment]
I think it would be great if the PPPC Utility could highlight the properties that are entitlement-related to an app selected in the list on the left.  This would at least draw attention to the properties that are more likely relevant to control for that app.

Sharing the app information (bundle ID, code signing info, entitlement info, etc) from other apps would definitely be something we could support.  We need to do a bit more coding to have our model understand that there can be entitlements associated to an app, and do the highlighting of associated properties.
